app/models/game.rb

```diff
@@ -1,7 +1,17 @@
 class Game < ActiveRecord::Base
+  attr_accessor :creator_plays_as_black # for checkbox access
```

Comment:
allows the checkbox in the game creation form to set this parameter and for it to be accessed in the controller